client/src/components/Dashboard/TaskItem/TaskItem.jsx

```diff
@@ -36,20 +41,19 @@ class TaskItem extends Component {
 
   deleteTaskHandle = (event) => {
     const { isDeleted } = this.state;
-    // const { taskListRef, id } = this.props;
+    const { taskDetailsActions, id } = this.props;
```

Comment:
You can destructure taskDetailsActions like this:
`const { taskDetailsActions: { deleteTaskDetails }, id } = this.props;`

and then use deleteTaskDetails directly:

`deleteTaskDetails(id);`